src/Symfony/Component/Worker/EventListener/ClearDoctrineListener.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Worker\EventListener;
+
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Worker\Consumer\ConsumerEvents;
+use Symfony\Component\Worker\Loop\LoopEvents;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class ClearDoctrineListener implements EventSubscriberInterface
+{
+    private $doctrine;
+
+    public function __construct(EntityManager $doctrine = null)
+    {
+        $this->doctrine = $doctrine;
```

Comment:
should probably be called `$em` or `$entityManager`. Any specific reason this is the implementation and not the interface?